src/Microsoft.Win32.Registry/src/Microsoft/Win32/RegistryKey.cs

```diff
@@ -634,6 +634,8 @@ public override string ToString()
         private static string FixupName(string name)
         {
             Debug.Assert(name != null, "[FixupName]name!=null");
+
+            // string.Contains(char) is .NetCore2.1+ specific
```

Comment:
@weshaggard is this correct, that M.Win32.R cannot use netcoreapp API? It builds for netcoreapp, but packages for netstandard.

Reply:
I will try...

Reply:
Oh wait. See @ahsonkhan's message above. That failed the build:
```
20:15:59 Microsoft\Win32\RegistryKey.cs(637,32): error CS1503: Argument 1: cannot convert from 'char' to 'string' [D:\j\workspace\windows-TGrou---33cbf18b\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
```